except_star.md

```diff
@@ -207,6 +250,8 @@ processed by one `except *` clause.
 
 ## Semantics
 
+### Overview
+
```

Comment:
I just red the text below carefully. Maybe you should call out that one big difference with classic try/except is that if there are multiple `except *` blocks, several of them may be executed? That's clear from the description, but I would still call it out as a key difference.